invenio_jobs/utils.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+from functools import wraps
+from time import sleep
+
+from invenio_db import db
+
+from invenio_jobs.models import Run, RunStatusEnum
+
+
+# TODO 1. Move to service? 2. Don't use kwargs?
+def update_run(run, **kwargs):
+    if not run:
+        return
+    for kw, value in kwargs.items():
+        setattr(run, kw, value)
+    db.session.commit()
+
+
+def is_a_job(func):
+    @wraps(func)
+    def wrapper(self, *args, **kwargs):
+        # TODO Add failsafe of if run doesn't exist
+        run = Run.query.filter_by(task_id=self.request.id).one_or_none()
+        update_run(run, status=RunStatusEnum.RUNNING, started_at=datetime.now())
```

Comment:
```suggestion
        update_run(run, status=RunStatusEnum.RUNNING, started_at=datetime.utcnow())
```

Ditto for other places where we take timestamps.